examples/org/workflowsim/examples/scheduling/DataAwareSchedulingAlgorithmExample2.java

```diff
@@ -0,0 +1,145 @@
+/**
```

Comment:
Could you change the algorithm name to be a real name, liek KClusteringSchedulingAlgorithmExample.java